src/store/index.js

```diff
@@ -91,6 +91,18 @@ export default new Vuex.Store({
 				return helper.logError(dispatch, 'transaction/initialize');
 			case 'statistics':
 				return helper.logError(dispatch, 'statistic/initialize');
+
+				// Page Detail
+			case 'account-detail':
+				return helper.logError(dispatch, 'account/fetchAccountDetail', route.params);
+			case 'block-detail':
+				return helper.logError(dispatch, 'block/fetchBlockInfo', route.params);
+			case 'mosaic-detail':
+				return helper.logError(dispatch, 'mosaic/fetchMosaicInfo', route.params);
+			case 'namespace-detail':
+				return helper.logError(dispatch, 'namespace/fetchNamespaceInfo', route.params);
+			case 'transaction-detail':
+				return helper.logError(dispatch, 'transaction/getTransactionInfoByHash', route.params);
```

Comment:
What is a purpose of this changes?

Reply:
To re-fetch data from new endpoint, after change the node in all those detail pages

Reply:
PageAssembler page has `initAction` property of the action name which dispatches on page load. Does it work?

Reply:
It only work in the listing page, but not detail page.

Reply:
It means there is a bug. This actions should be dispatched via PageAssembler (initAction prop). I will approve the PR, because this is a topic of another issue.